clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -102,7 +108,51 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
     this.namespace = namespace
   }
 
-  KubernetesServerGroup(ReplicaSet replicaSet, String account, List<Event> events, HorizontalPodAutoscaler autoscaler) {
+
+  KubernetesServerGroup(V1beta1DaemonSet daemonSet, String account, List<Event> events) {
+
+    this.name = daemonSet.metadata?.name
+    this.account = account
+    this.region = daemonSet.metadata?.namespace
+    this.namespace = this.region
+    this.createdTime = daemonSet.metadata?.creationTimestamp?.getMillis()
+    this.zones = [this.region] as Set
+    this.securityGroups = []
+    this.loadBalancers = KubernetesUtil.getLoadBalancers(daemonSet) as Set
+    this.launchConfig = [:]
+    this.labels = daemonSet.spec?.template?.metadata?.labels
+    this.deployDescription = KubernetesApiConverter.fromDaemonSet(daemonSet)
```

Comment:
@lwander  -  fromDaemonSet function is same like  all  functions in   KubernetesConvertor  Class  having functions    fromReplicaSet , from Container.   and etc.